libraries/browser-tracker-core/src/tracker/index.ts

```diff
@@ -180,12 +183,15 @@ export function Tracker(
         }
         return config.anonymousTracking?.withServerAnonymisation === true ?? false;
       },
-      getAnonymousTracking = (config: TrackerConfiguration) => !!config.anonymousTracking;
+      getAnonymousTracking = (config: TrackerConfiguration) => !!config.anonymousTracking,
+      isWebPageContextAvailable = trackerConfiguration?.contexts?.webPage ?? true;
 
     // Get all injected plugins
     browserPlugins.push(getBrowserDataPlugin());
-    if (trackerConfiguration?.contexts?.webPage ?? true) {
-      browserPlugins.push(getWebPagePlugin()); // Defaults to including the Web Page context
+    /* When including the Web Page context, we add the relevant internal plugins */
+    if (isWebPageContextAvailable) {
```

Comment:
Question: I am not sure what is the better approach here, but when comparing with the mobile trackers, we have two separate configuration properties for activating the screen context (same as webpage context) and for the mobile context (same as browser context). Why did you decide to have both under the same property?

It is quite theoretical, but I can imagine someone wanting the web page context for the page view ID and not wanting to spend the extra payload size on the browser context entity.

Reply:
Yeap fixed!
`contexts: { browser: true }`